### PR TITLE
[Cleanup] Remove orphaned outlets

### PR DESCRIPTION
**Pull request by lorcan**
Addresses https://github.com/emberjs/ember.js/issues/15876#issuecomment-348680323

### CODE REVIEW

File: packages/ember-glimmer/lib/component-managers/render.ts [hunk 1]
```diff
@@ -49,10 +49,6 @@ class SingletonRenderManager extends AbstractRenderManager {
       this._pushToDebugStack(`controller:${name} (with the render helper)`, env);
     }
 
-    if (dynamicScope.rootOutletState) {
-      dynamicScope.outletState = dynamicScope.rootOutletState.getOrphan(name);
```
Comment (lorcan) [inline packages/ember-glimmer/lib/component-managers/render.ts]: Is `rootOutletState` still needed at all?

**Comment by thoov** [comment]
@lorcan I believe for most of this we are going to need to create a new ENV variable that we can wrap this deprecated code in. Here is an example of what I am talking about: https://github.com/emberjs/ember.js/pull/15900/files. This will also require a PR into [ember-2-legacy repo](https://github.com/emberjs/ember-2-legacy). If you need any help with either of these 2 things let me know.

**Comment by lorcan** [comment]
Great, thanks @thoov, I'll try that now.

**Comment by lorcan** [comment]
Hi @thoov gave this another go with https://github.com/emberjs/ember.js/pull/15925, taking inspiration from the PR you shared. I'm happy to close this PR in favour of that one if you want to continue the discussion there :-)

**Comment by lorcan** [comment]
Closing in favour of https://github.com/emberjs/ember.js/pull/15925

**Pull request closed without merging**